sceptre/cli/drift.py

```diff
@@ -51,14 +51,16 @@ def drift_detect(ctx: Context, path: str):
     plan = SceptrePlan(context)
     responses = plan.drift_detect()
 
+    output_format = "yaml" if context.output_format == "text" else context.output_format
```

Comment:
It might be clearer to do this instead:
```suggestion
    output_format = "json" if context.output_format == "json" else "yaml"
```